Setup/ericomshield-setup.sh

```diff
@@ -622,8 +619,12 @@ function pull_images() {
             arr=($line)
             if [ $LINE -ge 3 ]; then
                 echo "################## Pulling images  ######################"
-                echo "pulling image: ${arr[1]}"
-                docker pull "securebrowsing/${arr[1]}"
+                echo "pulling image: ${arr[1]} ($SHIELD_REGISTRY)"
+                if [ ! -z "$SHIELD_REGISTRY" ]; then
+                   IMAGE_NAME="$SHIELD_REGISTRY/securebrowsing/${arr[1]}"
```

Comment:
If we want to use a local registry we also need to have this $SHIELD_REGISTRY address in the yaml file as Docker does not support changing default registry address: https://github.com/moby/moby/issues/33069

Reply:
As discussed it seems to work after we pull the image with the registry IP
If we see that it is required we will add the registry ip to the yml file